src/RA_AchievementOverlay.cpp

```diff
@@ -751,7 +749,7 @@ void AchievementOverlay::DrawNewsPage(HDC hDC, int nDX, _UNUSED int, const RECT&
 
     HGDIOBJ hOldObject = SelectObject(hDC, g_hFontDesc2);
 
-    for (int i = m_nNewsSelectedItem; i < static_cast<int>(m_LatestNews.size()); ++i)
+    for (int i = m_nNewsSelectedItem; i < gsl::narrow<int>(m_LatestNews.size()); ++i)
```

Comment:
#221 has a different (more efficient) way of doing this but just wanted to get rid of the warning.